src/box/tuple_format.c

```diff
@@ -166,12 +166,21 @@ tuple_field_delete(struct tuple_field *field)
 
 /** Return path to a tuple field. Used for error reporting. */
 static const char *
-tuple_field_path(const struct tuple_field *field)
+tuple_field_path(const struct tuple_field *field,
+                 const struct tuple_format *format)
```

Comment:
Mess with tabs and spaces. Should be 2tabs+1space to alignt const words.

Reply:
Fix